zulipterminal/model.py

```diff
@@ -2071,3 +2071,17 @@ def poll_for_events(self) -> None:
                         self.controller.raise_exception_in_main_thread(
                             sys.exc_info(), critical=False
                         )
+
+    def show_guest_suffix(self, user_id: int) -> bool:
+        """
+        Returns True if the realm_enable_guest_user_indicator
+        setting is enabled and the user is a guest
+        """
+        user = self.get_user_info(user_id)
+        assert user is not None
```

Comment:
Based on existing model code, let's raise a RuntimeError here.

We should move towards better exceptions for these cases in future though.